tools/webpack.config.ts

```diff
@@ -184,6 +184,9 @@ export default (env?: string): webpack.Configuration => {
             watchOptions: {
                 poll: 100,
             },
+            headers: {
+                "Access-Control-Allow-Origin": "http://localhost:9991",
+            },
```

Comment:
Can you explain why we need this change?  We might need to also have something related to `*.zulipdev.com`; otherwise realms you create with the "create realm" button won't work.

(Probably worth splitting to its own commit as well)

Reply:
The network request made in https://github.com/zulip/zulip/pull/12608/files#diff-128e0920d73303bbd167e336de1ab806R169 is to `localhost:9994` but the script is running from `localhost:9991`, and different ports are considered different origins by the same-origin policy.

> We might need to also have something related to *.zulipdev.com

I'll have a look into that

Reply:
Turns out `Access-Control-Allow-Origin` only allows the value `*`, `null` and an exact address, no regex or multiple addresses. What should I do here?

Reply:
We already have a proxy for webpack port; request made to `localhost:9991/webpack` goes to the webpack port. So just replacing  `localhost:9994` to `localhost:9991/webpack` should work? (Don't have time to check right now though)

Reply:
The /webpack path only have the built bundles, not the file system. Or maybe I am missing something?

Reply:
Yeah, never mind you are right!

Reply:
I think the right solution is we should add something in `zproject/dev_urls.py` to expose that part of the filesystem to the web, and then access that URL via port 9991?

Reply:
@timabbott yeah that sounds good.